src/main/java/gift/entity/Wish.java

```diff
@@ -0,0 +1,25 @@
+package gift.entity;
+
+public class Wish {
+    private Long id;
+    private Long memberId;
+    private Long productId;
```

Comment:
Wish의 상태 값들을 `final`로 선언해도 되지 않을까요? 😃

```suggestion
    private final Long id;
    private final Long memberId;
    private final Long productId;
```